lib/assessment.rb

```diff
@@ -43,6 +43,6 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    File.open(filename).reduce('') { |memo, line| memo + line.chomp + ' ' }.strip
```

Comment:
Actually, never mind. I forgot that the last line will also end in a special character, EOF; as such, `.chomp` will remove that instead.